### ISSUE TITLE
Problem encrypting using accents

### ISSUE THREAD

**Issue by ioapps-dev** [srikanth-lingala/zip4j#283]
**The library allows you to encrypt with accents, however it fails when trying to decrypt with the same password.**
Example: try to decrypt using the password 'útil'

```
Caused by: net.lingala.zip4j.exception.ZipException: Wrong Password
        at net.lingala.zip4j.crypto.AESDecrypter.init(AESDecrypter.java:76)
        at net.lingala.zip4j.crypto.AESDecrypter.<init>(AESDecrypter.java:50)
        at net.lingala.zip4j.io.inputstream.AesCipherInputStream.initializeDecrypter(AesCipherInputStream.java:37)
        at net.lingala.zip4j.io.inputstream.AesCipherInputStream.initializeDecrypter(AesCipherInputStream.java:18)
        at net.lingala.zip4j.io.inputstream.CipherInputStream.<init>(CipherInputStream.java:24)
        at net.lingala.zip4j.io.inputstream.AesCipherInputStream.<init>(AesCipherInputStream.java:32)
        at net.lingala.zip4j.io.inputstream.ZipInputStream.initializeCipherInputStream(ZipInputStream.java:196)
        at net.lingala.zip4j.io.inputstream.ZipInputStream.initializeEntryInputStream(ZipInputStream.java:186)
        at net.lingala.zip4j.io.inputstream.ZipInputStream.getNextEntry(ZipInputStream.java:103)
        at net.lingala.zip4j.io.inputstream.ZipInputStream.getNextEntry(ZipInputStream.java:77)
```

Thank you !!

**Comment by srikanth-lingala** [comment]
Which version of zip4j are you using? 
This might be a duplicate of #282 which is fixed already and will be included in the next release. Can you test it with `2.6.5-SNAPSHOT` and see if it works? If you have trouble accessing the snapshot version, please check [this][1] answer.

[1]: https://stackoverflow.com/questions/7715321/how-to-download-snapshot-version-from-maven-snapshot-repository

**Comment by ioapps-dev** [comment]
Hello, I am using version 2.6.1, I will try the version that you have proposed.
I will confirm if it works, thanks !!

**Comment by ioapps-dev** [comment]
I have tried Arabic letters and accents, this version works perfectly.
Thanks again, regards.

**Comment by srikanth-lingala** [comment]
Issue fixed in v2.7.0 which was released today.
